weave/trace/weave_client.py

```diff
@@ -1325,6 +1391,36 @@ def fail_call(self, call: Call, exception: BaseException) -> None:
         """Fail a call with an exception. This is a convenience method for finish_call."""
         return self.finish_call(call, exception=exception)
 
+    @trace_sentry.global_trace_sentry.watch()
+    def call_descendants(
+        self, call: Call, depth: int | None = None, limit: int | None = None
+    ) -> list[Call]:
+        """Get all descendant calls of a given call."""
+        res = self.server.calls_descendants(
+            CallsDescendantsReq(
+                project_id=self._project_id(),
+                call_ids=[call.id],
+                depth=depth,
+                limit=limit,
+            )
+        )
+        return list(res)
+
+    @trace_sentry.global_trace_sentry.watch()
+    def calls_descendants(
+        self, call_ids: list[str], depth: int | None = None, limit: int | None = None
+    ) -> list[Call]:
+        """Get all descendant calls of a given list of call IDs."""
+        res = self.server.calls_descendants(
+            CallsDescendantsReq(
+                project_id=self._project_id(),
+                call_ids=call_ids,
+                depth=depth,
+                limit=limit,
+            )
+        )
+        return list(res)
```

Comment:
I would collapse into the previous function.  Instead, you can have it take flexible inputs and do the handling inside the func
```py
CallId = str
def call_descendants(self, call_ids: Call | CallId | Iterable[Call | CallId], ...): ...
```